lib/SimpleHttpClient/SslClientConnection.cpp

```diff
@@ -309,11 +313,35 @@ bool SslClientConnection::connectSocket() {
     return false;
   }
 
+  if (_isSocketNonBlocking) {
```

Comment:
How is this supposed to work on Windows? Windows does not have `fcntl`. Therefore the jenkins build on Windows does not work. Does Windows even have non-blocking sockets?

Reply:
That can be done on windows by 
`unsigned long mode = 1; ioctlsocket(fd, FIONBIO, &mode);`